themes/default/content/docs/concepts/inputs-outputs/_index.md

```diff
@@ -509,7 +825,7 @@ let certValidation = new aws.route53.Record("cert_validation", {
 
 ## Working with Outputs and Strings {#outputs-and-strings}
 
-Outputs that contain strings cannot be used directly in operations such as string concatenation. String interpolation lets you more easily build a string out of various output values, without needing {{< pulumi-apply >}} or [Output.all](/docs/reference/pkg/python/pulumi#pulumi.Output.all). You can use string interpolation to export a stack output, provide a dynamically computed string as a new resource argument, or even for diagnostic purposes.
+Outputs that resolve to strings cannot be used directly in operations such as string concatenation until the output has been resolved. In these scenarios, you'll need to resolve the values using {{< pulumi-apply >}}.
```

Comment:
I think some of the removed text was actually very helpful for suggesting what you should use for common string manipulation options over outputs. The new text suggests you should “reaolve” to a plain string, which I think a lot of users will want to interpret as awaiting the output, then using the resulting value. This is exactly what you *cant* do in Pulumi, but which I worry we are suggesting with the “resolve” terminology.